content/shmem_team_broadcast.tex

```diff
@@ -0,0 +1,90 @@
+\apisummary{
+    Broadcasts a block of data from one \ac{PE} to one or more destination
+    \acp{PE}.
+}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void @\FuncDecl{shmem\_team\_broadcast32}@(shmem_team_t team, void *dest, const void *source, size_t nelems, int PE_root);
+void @\FuncDecl{shmem\_team\_broadcast64}@(shmem_team_t team, void *dest, const void *source, size_t nelems, int PE_root);
+\end{Csynopsis}
+
+\begin{apiarguments}
+
+\apiargument{IN}{team}{A valid \openshmem team handle to a team which has been created without
+    disabling support for collective operations.}
+\apiargument{OUT}{dest}{A symmetric data object. See the table below in this description
+    for allowable types} 
+\apiargument{IN}{source}{A symmetric data object that can be of any data type
+    that is permissible for the \dest{} argument.}
+\apiargument{IN}{nelems}{The number of elements in \source{}.  For
+    \FUNC{shmem\_team\_broadcast32}, this is the number of
+    32-bit halfwords. nelems must be of type \CTYPE{size\_t} in \Cstd.}
+\apiargument{IN}{PE\_root}{Zero-based ordinal of the \ac{PE}, with respect to
+    the team, from which the data is copied. Must be greater than or equal to
+    0 and less than N-1, where N is the result of calling \FUNC{shmem\_team\_n\_pes(team)}.
+    \VAR{PE\_root} must be of type \CTYPE{int}.} 
+
+\end{apiarguments}
+
+\apidescription{   
+    \openshmem broadcast routines are collective routines over an existing team.
+    They copy data object \source{} on the processor specified by \VAR{PE\_root}
+    and store the values at \dest{} on the other \acp{PE} that are members of the
+    team. The data is not copied to the \dest{} area on the root \ac{PE}.
+    
+    If the team has been created with the \LibConstRef{SHMEM\_TEAM\_NOCOLLECTIVE} option,
+    it will not have the required support structures to complete this routine. If
+    such a team is passed to this or any other team collective routine, the behavior
+    is undefined.
+
+    As with all \openshmem routines where the operation occurs for a given team -
```

Comment:
Alternatively - `As with all OpenSHMEM team collective routines, each of these routines assumes that only PEs in the team call the routine. If a PE not in the team calls an OpenSHMEM team collective routine, the behavior is
undefined.`

Reply:
I agree that the current text is convoluted and can be simplified. I want to make some mention that PE numbers are relative to the team in descriptions throughout the document, just to make sure it is very obvious, but restating the range requirements is too much.